ONSdigital/ssdc-rm-acceptance-tests
# 3352 split acceptance tests into a lightweight core set and full regression set

# Motivation and Context
Add regression tags

# What has changed
Added regression tags and new commands etc

# How to test?
Run locally:  make test = run all,  make test_core = run none regression tests only.   Also should show skipped now.

# Links
https://trello.com/c/c9ZR6lt4/3352-split-acceptance-tests-into-a-lightweight-core-set-and-full-regression-set-8
